app/views/content_items/statistical_data_set.html.erb

```diff
@@ -1,7 +1,41 @@
-<%= render 'shared/title_and_translations', content_item: @content_item %>
-<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
-<%= render 'shared/metadata', content_item: @content_item %>
-<%= render 'shared/history_notice', content_item: @content_item %>
-<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
-<%= render 'shared/sidebar_contents_with_body', content_item: @content_item %>
-<%= render 'shared/footer', @content_item.document_footer %>
+<%= content_for :simple_header, @navigation.should_present_taxonomy_navigation? %>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
```

Comment:
I think indentation should be set to 2 spaces for templates, looks to be the case in others.

Reply:
Another case for ERB linting.